Generals/Code/GameEngine/Source/GameClient/GUI/ControlBar/ControlBarObserver.cpp

```diff
@@ -89,6 +89,7 @@ static NameKeyType buttonCancelID = NAMEKEY_INVALID;
 
 static GameWindow *winFlag = NULL;
 static GameWindow *winGeneralPortrait = NULL;
+static GameWindow *buttonIdleWorker = NULL;
```

Comment:
Maybe add a comment here or somewhere appropriate that the idle worker button is now also presented in the Observer Bar.

Reply:
Can I push to this branch? Then I can tweak that.